src/components/FormAlertWithSubmitButton.js

```diff
@@ -63,25 +68,30 @@ const FormAlertWithSubmitButton = props => (
         message={props.message}
         onFixTheErrorsLinkPressed={props.onFixTheErrorsLinkPressed}
     >
-        {isOffline => ((isOffline && !props.enabledWhenOffline) ? (
-            <Button
-                success
-                isDisabled
-                text={props.buttonText}
-                style={[styles.mb3]}
-                danger={props.isSubmitActionDangerous}
-            />
-        ) : (
-            <Button
-                success
-                pressOnEnter={!props.disablePressOnEnter}
-                text={props.buttonText}
-                onPress={props.onSubmit}
-                isDisabled={props.isDisabled}
-                isLoading={props.isLoading}
-                danger={props.isSubmitActionDangerous}
-            />
-        ))}
+        {isOffline => (
+            <View>
+                {(isOffline && !props.enabledWhenOffline) ? (
```

Comment:
We don't need to check isOffline here since it's implied from before

Reply:
no right? isOffline can be true or false. Am I misunderstanding the question here?

Reply:
Oh nevermind, thought this was a conditional render